### PR TITLE
Ensure websocket polyfill doesn't load if there is already a `global.Webocket` property set

### PR DESCRIPTION
**Pull request by Jay2theWhy**
## Description
A quick conditional check for global.WebSocket. Similar issue to this: https://github.com/vercel/next.js/issues/41866 which was resolved with the following PR: https://github.com/spacebudz/lucid/pull/108/commits/b550ebc8b0bcb9399b6df0eefff172ecad81a99e

Closes: #5583 

## Tests
All tests passed:
image

### CODE REVIEW

File: client/js/src/client.ts [hunk 1]
```diff
@@ -259,7 +259,7 @@ export function api_factory(
 				const ws = await import("ws");
 				NodeBlob = (await import("node:buffer")).Blob;
 				//@ts-ignore
-				global.WebSocket = ws.WebSocket;
+				if (!global.WebSocket) global.WebSocket = ws.WebSocket;
```
Comment (pngwn) [inline client/js/src/client.ts:262]: Could we live this check to line 258 instead? If we don't need the poly fill then we shouldn't bother loading it in the first place.

**Review by pngwn — COMMENTED** [review]
Left a small comment. What I am curious about is why websockets exist in a non-browser environment, is the situation here that a mock or polyfill has already been loaded?

File: client/js/src/client.ts (hunk 1 above)
Comment (Jay2theWhy) [reply client/js/src/client.ts]: Good call. Changed 👍

**Review by pngwn — APPROVED** [review]
This is great. Thanks for taking care of this @Jay2theWhy!

**Pull request merged**